routers/web/repo/repo.go

```diff
@@ -158,8 +157,17 @@ func Create(ctx *context.Context) {
 		}
 	}
 
-	ctx.Data["CanCreateRepo"] = ctx.Doer.CanCreateRepo()
+	canCreateRepo := ctx.Doer.CanCreateRepo()
+	if !canCreateRepo && ctx.Doer.ID == ctxUser.ID {
+		orgs, has := ctx.Data["Orgs"].([]*organization.Organization)
+		if has && len(orgs) > 0 {
+			ctxUser = orgs[0].AsUser()
```

Comment:
Why is the first organizaiton?

Reply:
Because curent user can't create repo, and user hasn't choose an organizaiton by query option, so use the the first organizaiton as default.